rtv/terminal.py

```diff
@@ -438,6 +439,20 @@ def open_editor(self, data=''):
             else:
                 _logger.info('File deleted: %s', filepath)
 
+    def open_urlview(self, data):
+        urlview = os.getenv('URLVIEWER') or os.getenv('RTV_URLVIEWER') \
```

Comment:
Is ``$URLVIEWER`` is a globally adopted variable (like ``$BROWSER`` and ``$PAGER``)? I did a quick google search and couldn't find anything. Ideally we would be able to use something standard, but if nothing exists then I think this is ok.